docs/tutorial/recent-documents.md

```diff
@@ -27,6 +27,32 @@ const { app } = require('electron')
 app.clearRecentDocuments()
 ```
 
+
+__Adding Recent documents in Menuitem (For Mac):__
```

Comment:
I would suggest putting this under "macOS notes".

Reply:
Yes, That's a better idea!!